hackerrank/1-week-prep/day-6/simple-text-editor.py

```diff
@@ -0,0 +1,58 @@
+class stack:    
```

Comment:
As per [documentation](https://realpython.com/python-pep8/):
```suggestion
class Stack:    
```